superset/connectors/sqla/models.py

```diff
@@ -883,9 +883,9 @@ def get_sqla_query(  # sqla
                     elif op == utils.FilterOperationType.LIKE.value:
                         where_clause_and.append(col_obj.get_sqla_col().like(eq))
                     elif op == utils.FilterOperationType.IS_NULL.value:
-                        where_clause_and.append(col_obj.get_sqla_col() is None)
+                        where_clause_and.append(col_obj.get_sqla_col() == None)
```

Comment:
`is None` and `is not None` is preferred

Reply:
It actually doesn't work in this case, SqlAlchemy expects them to be in this form.

Reply:
Going to add unit tests.

Reply:
oh yes I understand!